styrdokument/Datavetenskapsdivisionen stadga.tex

```diff
@@ -352,10 +336,11 @@
 		\begin{enumerate}[label=\theenumi~\alph* §, ref=\theenumi~\alph*]
 			\item Vid ordförandes avgång eller avstättning ska vice ordförande ta upp ordförandes arbetsuppgifter och rättigheter, tills dess att divisionsstämman utser en ny ordförande.
 
-			\item Vid styrelsens avsättning ska en temporär styrelse väljas på innevarande sammanträde.
-			Den temporära styrelsen tar över ordinarie styrelses befogenheter och skyldigheter tills dess en ny ordinarie styrelse har valts.
-			Den temporära styrelsen eller talmanspresidiet ska annonsera ett extra sammanträde av divisionsstämman.
-			Sammanträdet ska ske inom 15 läsdagar eller sista läsdag för terminen, vad som kommer först.
+			\item 
+		Vid styrelsens avsättning ska en temporär styrelse väljas på innevarande sammanträde.
+		Den temporära styrelsen tar över ordinarie styrelses befogenheter och skyldigheter tills dess en ny ordinarie styrelse har valts.
+		Den temporära styrelsen eller talmanspresidiet ska annonsera ett extra sammanträde av divisionsstämman.
+		Sammanträdet ska ske inom 15 läsdagar eller sista läsdag för terminen, vad som kommer först.
```

Comment:
Du har råkat klistra in den gamla lydelsen av paragrafen. Indentering och flytta upp en rad.